### PR TITLE
Make RPC concurrency configurable

### PR DESCRIPTION
**Pull request by omerfirmak**

### CODE REVIEW

File: cmd/juno/juno.go [hunk 1]
```diff
@@ -68,6 +69,7 @@ const (
 	defaultMetricsPort         = 9090
 	defaultGRPC                = false
 	defaultGRPCPort            = 6064
+	defaultRPCMaxConcurrency   = 1024
```
Comment (joshklop) [inline cmd/juno/juno.go:72]: Is `1024` better than `2 * runtime.GOMAXPROCS(0)`?

**Review by joshklop — APPROVED** [review]
Will users see benefits from tuning this new knob? My gut feeling is that we can set a sane default for all cases, but maybe I'm missing something.

File: cmd/juno/juno.go (hunk 1 above)
Comment (omerfirmak) [reply cmd/juno/juno.go:72]: 1024 is practically unlimited concurrency. Users can decide to limit if they want to.

@wojciechos was seeing `broken pipe` errors in load tests. I am assuming those were related to `2 * runtime.GOMAXPROCS(0)` being too low

**Comment by omerfirmak** [comment]
> Will users see benefits from tuning this new knob? My gut feeling is that we can set a sane default for all cases, but maybe I'm missing something.

HW specs will vary quite a lot, I dont think we can find a good default that will be good for the entire range of HW that users will run Juno on.

**Pull request merged**